yt_dlp/extractor/tenplay.py

```diff
@@ -62,27 +86,44 @@ def _get_bearer_token(self, video_id):
 
     def _real_extract(self, url):
         content_id = self._match_id(url)
-        _token = self._get_bearer_token(content_id)
         data = self._download_json(
             'https://10play.com.au/api/v1/videos/' + content_id, content_id)
+        headers = {}
+
+        if data.get('memberGated') is True:
+            _token = self._get_bearer_token(content_id)
+            headers = {'Authorization': _token}
+
         _video_url = self._download_json(
             data.get('playbackApiEndpoint'), content_id, 'Downloading video JSON',
-            headers={'Authorization': _token}).get('source')
+            headers=headers).get('source')
         m3u8_url = self._request_webpage(HEADRequest(
             _video_url), content_id).geturl()
         if '10play-not-in-oz' in m3u8_url:
             self.raise_geo_restricted(countries=['AU'])
         formats = self._extract_m3u8_formats(m3u8_url, content_id, 'mp4')
         self._sort_formats(formats)
 
+        subtitles = {}
+        caption_url = data.get('captionUrl')
+        if caption_url:
+            subtitles.setdefault('en', []).append({
+                'url': caption_url,
+                'ext': determine_ext(caption_url, None),
+            })
+
         return {
             'formats': formats,
+            'subtitles': subtitles,
```

Comment:
```suggestion
            'subtitles': {'en': [{'url': data.get['captionUrl']}]} if data.get('captionUrl') else None,
```
`determine_ext` will be applied by core code